packages/babel-plugin-proposal-private-methods/test/fixtures/static-accessors-modern/basic/output.js

```diff
@@ -0,0 +1,22 @@
+class Cl {
+  static #privateStaticFieldValue = Object.defineProperty({
+    t: this
+  }, "_", {
+    get: function () {
+      return Cl.#PRIVATE_STATIC_FIELD;
+    },
+    set: function (newValue) {
+      Cl.#PRIVATE_STATIC_FIELD = `Updated: ${newValue}`;
+    }
+  });
```

Comment:
The trick here is to rely on native getters/setters. By doing so we don't have to worry about replacing `this.#foo`, `this.#foo += 2`, etc. with complex function calls, since we can just replace `this.#foo` with `this.#foo._`.